Microsoft.Toolkit.Uwp.UI.Controls.Input/ColorPicker/ColorPickerSlider.cs

```diff
@@ -297,5 +299,14 @@ private void OnDependencyPropertyChanged(object sender, DependencyPropertyChange
 
             return;
         }
+
+        /// <summary>
+        /// Event handler for when the control's theme changes.
+        /// </summary>
+        private void ColorPickerSlider_ActualThemeChanged(FrameworkElement sender, object args)
+        {
+            this.UpdateColors();
```

Comment:
I see this getting called, but the UpdateColors is returning the same value in both cases, so I see no functional change to the result that it's still hard to see the transparency in the switch (unless I reload the sample).

This also effects the shade boxes in the parent control, so they'd need an update too, eh?

Reply:
I'm not sure where you are going with this. But as stated it works as expected. Notice the color slider color is actually changing from white to black when the background changes with the theme. It has nothing to do with the checkered background or anything else.

![Animation](https://user-images.githubusercontent.com/17993847/197080759-596d77ce-0390-47ba-8df2-c78873fe4717.gif)

Reply:
Thanks for the clarification @robloo, I misunderstood what the issue and fix was about, I do see this resolves the issue now.